docs/_posts/2019-03-05-go.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Getting GOing"
+date: 2019-03-05 09:00:00 -0600
+author: Sigal Sax
+categories: blog
+published: true
+image: ship_it.jpg
+thumb: ship_it.jpg
+image-alt: Test post
+excerpt: "For the past year, we on the Cyberark Conjur team have been working on our latest open source project in Go..."
```

Comment:
since this is the preview that will show on the Secretless blog page, can we revise it to be something that explains more about what it will discuss? something about our experience using Go for Secretless, maybe? the current wording sounds like we're introducing the project

Reply:
Thought of the following:
"Why did we choose Go for our latest open source project, the Secretless Broker? Go gave us the perks of a low-level language while providing us with a toolset and native capabilities of a higher-level one."
OR
"Why we chose Go for our latest open source project, the Secretless Broker and how this decision has impacted our development process"

Reply:
I like the second one better 👍